### DISCUSSION TITLE
Different sort types in Author view

### DISCUSSION THREAD

**Discussion by WhatsAUserName** [LeoKlaus/plappa#198] [Feature Requests]
Hi! Was wondering how feasible it would be to add custom ordering in the author page, currently it’s alphabetical which isn’t always what works best for my specific use case. Sorting by year or series (like on the ABS web interface) would be great if possible!

**Comment by Cantello** [comment]
Additionally,  sort options in the 'Series' tab would be great. Currently it appears to be random, neither alphabetically, nor by e.g. year. In the web-ui, it is automatically sorted alphabetically, only backwards, which works, as newest episodes (with higher numbers) are first in the list.
